src/main/java/io/github/opencubicchunks/cubicchunks/chunk/cube/EmptyCube.java

```diff
@@ -1,13 +1,63 @@
 package io.github.opencubicchunks.cubicchunks.chunk.cube;
 
 import io.github.opencubicchunks.cubicchunks.chunk.util.CubePos;
+import net.minecraft.core.BlockPos;
+import net.minecraft.server.level.ChunkHolder;
 import net.minecraft.world.level.Level;
+import net.minecraft.world.level.block.Blocks;
+import net.minecraft.world.level.block.entity.BlockEntity;
+import net.minecraft.world.level.block.state.BlockState;
 import net.minecraft.world.level.chunk.LevelChunkSection;
+import net.minecraft.world.level.material.FluidState;
+import net.minecraft.world.level.material.Fluids;
+import org.jetbrains.annotations.Nullable;
 
 public class EmptyCube extends BigCube {
 
-    public EmptyCube(Level worldIn) {
-        super(worldIn, CubePos.of(0, 0, 0), null);
+    public EmptyCube(Level worldIn, CubePos cubePos) {
```

Comment:
This change looks like there are cases where we need the position of an empty cube. Can you give an example?

Reply:
This matches the constructor of the vanilla `EmptyChunk` so we should aswell.